components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -205,4 +205,14 @@ - (void)contentDrawerTopInset:(CGFloat)transitionToTop {
   [self.delegate bottomDrawerControllerDidChangeTopInset:self topInset:topInset];
 }
 
+- (void)expandToFullScreenWithDuration:(NSTimeInterval)duration
+                            completion:(void (^)(BOOL))completion {
+  if ([self.presentationController isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
+    MDCBottomDrawerPresentationController *bottomDrawerPresentationController =
+        (MDCBottomDrawerPresentationController *)self.presentationController;
+    [bottomDrawerPresentationController expandToFullScreenWithDuration:duration];
+  }
+  completion(YES);
```

Comment:
how are you assuring the completion block is only called after the animation is complete? the animation is done asynchronously, and this will be called immediately after the invocation. Please pass the completion block further in and put it where the completion of the animation block is.